### ISSUE TITLE
Limit image size breaking retina is confusing

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/optimole-wp#787]
### What problem does this address?

I couldn't understand why images on Codeinwp are not looking great and I have realised this is because retina is disabled. It was hard to detect because in fact, the setting was enabled, just that I noticed after in the fine print of limit image size, which is enabled by default now, that Retina doesn't work with that.

If that setting is coming on by default, I don't think anyone would read through the whole paragraph to see that retina images aren't working, considering they can just enable that before.

https://vertis.d.pr/i/F138SN

### What is your proposed solution?

If we want to leave the settings by default as they are now, I suggest that when a user tries to enable retina we alert them that we disabled limit image size setting for that to work.

Also if the image size is disabled and the user enables it while retina is also active, we alert them that we need to disable retina, since those can't function together.

Basically don't let the user believe that they can enable both of them at the same time.

### Will this feature require documentation? (Optional)

None

**Comment by kushh23** [comment]
We have something similar reported here - https://github.com/Codeinwp/optimole-wp/issues/767
